tests/mocha/server/RoundTest.js

```diff
@@ -41,9 +49,22 @@ MochaWeb.testOnly(function() {
       chai.expect(open.isClosed()).to.be.false;
       chai.expect(closed.isClosed()).to.be.true;
     });
+
+    it('displayTitle()', function() {
+      chai.expect(makeRound({eventCode: 'skewb', nthRound: 1, totalRounds: 2}).displayTitle()).to.equal('Skewb: Round 1 of 2');
+      chai.expect(makeRound({eventCode: 'skewb', nthRound: 1, totalRounds: 1}).displayTitle()).to.equal('Skewb: Single Round');
+    });
+
+    it('isLast()', function() {
+      chai.expect(makeRound({nthRound: 1, totalRounds: 2}).isLast()).to.be.false;
+      chai.expect(makeRound({nthRound: 2, totalRounds: 2}).isLast()).to.be.true;
+    });
   });
 
   function makeRound(properties) {
-    return Rounds.findOne(Rounds.insert(_.extend({ competitionId: 'fake'}, properties)));
+    return Rounds.findOne(Rounds.insert(_.extend({ competitionId: 'fake', nthRound: 1, totalRounds: 1 }, properties)));
```

Comment:
It's too bad that these `make*` methods have to be copied around to each test. This is exactly the thing that `FactoryGirl` over in rails gives you, right? Perhaps it would make sense to create our own FactoryGirl lite? Basically just consolidate these methods in a `Factory` namespace or something?